src/fswAlgorithms/attControl/thrMomentumManagementCpp/_UnitTest/test_thrMomentumManagementCpp.py

```diff
@@ -99,6 +99,76 @@ def test_thrMomentumManagement(show_plots, hsMinCheck):
     np.testing.assert_allclose(trueVector, dataLog.torqueRequestBody, rtol=accuracy)
 
 
+@pytest.mark.parametrize("hd_B", [np.array([1.0, 0.0, 0.0])])
```

Comment:
:small_red_triangle_down: With one value there's no need to parameterize this test. We can set the value directly from within the test.

Reply:
Done.

Reply:
No quite. Need to remove`hd_B` from the function prototype.

Reply:
Ah, I took it out of one but not the other. It should be fixed now.